src/python/pants/backend/native/subsystems/native_build_step.py

```diff
@@ -35,8 +35,8 @@ def register_options(cls, register):
              help='The default for the "compiler_option_sets" argument '
                   'for targets of this language.')
 
-    register_enum_option(
-      register, ToolchainVariant, '--toolchain-variant', advanced=True, default='gnu',
+    ToolchainVariant.register_option(
```

Comment:
This works, but I think it's seen as better style to do `cls.register_option`, because it makes it more immediately clear this method belongs to this class rather than another class.

Reply:
I have changed all of these to normal `register()` calls!